server/src/main/resources/routes

```diff
@@ -41,8 +41,7 @@ GET /admin/users/:userId/logs controllers.AdminController.getUserLogs(userId)
 # get frontend configs
 GET /environment-config controllers.EnvironmentConfigController.getEnvironmentConfig()
 
-# static assets for swagger-ui
-GET /docs/*file        controllers.Assets.at(path="/swagger-ui", file)
+-> / controllers.ApiRoutes
```

Comment:
Existing routes must be server through swagger

Reply:
Added a TODO for next PR